git-extra/astextplain

```diff
@@ -12,7 +12,14 @@ case "$1" in
 		odt2txt "$1" || cat "$1"
 		;;
 	*.doc | *.DOC | *.dot | *.DOT)
-		out=$(antiword -m UTF-8 "$1") && sed "s/\^M$//" <<<$out || cat "$1"
+		case "$(file --brief --mime-type "$1")" in
```

Comment:
My first reaction was to be concerned about the time spent on spawning `file`, but [`file` has hard enough of a time to identify all the Word Document flavors](https://github.com/search?q=repo%3Afile%2Ffile%20msword&type=code) that the alternative (namely, to hand-roll a script to identify Word Documents) seems prohibitive.

My next reaction was to be concerned about MinGit, but then I looked and found out that MinGit does not ship with `astextplain` (nor with `file`, for that matter).

Reply:
'